SL/Form.pm

```diff
@@ -392,17 +398,35 @@ qq|<meta http-equiv="Content-Type" content="text/plain; charset=$self->{charset}
   <title>$self->{titlebar}</title>
   <meta name="robots" content="noindex,nofollow" />
   $favicon
+
+  <link rel="stylesheet" href="css/select2-4.0.13.min.css" type="text/css"/>
+  <link rel="stylesheet" href="css/jquery-ui-1.12.1.min.css" type="text/css"/>
+
   $stylesheet
+
   $charset
 
   <script src="js/jquery-3.6.0.min.js" type="text/javascript"></script>
-  <script src="js/jquery-ui-1.8.6.custom.min.js" type="text/javascript"></script>
+  <script src="js/jquery-ui-1.12.1.min.js" type="text/javascript"></script>
```

Comment:
mhhh... before some custom was used, was there some custom js from us in it? Or why custom?

Reply:
Will have to check

Reply:
@stoldorma  I would DL the 1.8.6 and compare it with the one from the repo

Reply:
It's the same js file, the only difference is one added ; at the end in the ledger version.

Note that the ledger uses this js file:
Download ([jQuery UI 1.8.6 (source, theme, demos, tests)](https://jqueryui.com/resources/download/jquery-ui-1.8.6.zip)) -> unzip -> ui -> minified -> jquery.effects.core.min.js

Reply:
https://jqueryui.com/download/all/

Reply:
And why did we wanted to use latest jquery UI? Is is required for the autofill dropdowns? Same as in the armadan's PR?

Reply:
I think I took it from there, yes (or something didn't work). I don't remeber, sry